bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/values/TableValueImpl.java

```diff
@@ -585,14 +585,14 @@ public V getData(K key) {
             if (entryList == null) {
                 return null;
             }
-            if (entryList.size() > 1) {
+            if (entryList.size() >= 1) {
```

Comment:
Can we not simply remove this `if` condition?

Reply:
+1